routes/auth.js

```diff
@@ -60,7 +60,7 @@ setup.getStrategy = () => {
     clientSecret: config.auth.github.clientSecret,
     // this needs to match the callback url on the oauth app on github
     callbackURL: `${config.endpoints.service}/auth/github/finalize`,
-    scope: ['repo', 'user']
+    scope: ['public_repo', 'read:user', 'read:org']
```

Comment:
Nice to scope those down. did you confirm that public_repo gives the user enough perms to open a pull request?

Reply:
Yep, it functioned fine when I last tested it. Latest pull of web UI isn't quite functioning for me at the moment (missing navbar & curation screen non-functioning) but it looks like you were in the middle of something there.